package.json

```diff
@@ -18,10 +18,10 @@
     "test": "test"
   },
   "scripts": {
-    "chain": "yarn clean-chain && ganache-cli --db blockchain --networkId 50 --accounts 20 -l 19000000 -e 10000000000 -m 'concert load couple harbor equip island argue ramp clarify fence smart topic'",
+    "chain": "yarn clean-chain && ganache-cli --db blockchain --networkId 50 --accounts 20 -l 19000000 -e 10000000000 -m 'concert load couple harbor equip island argue ramp clarify fence smart topic' --allowUnlimitedContractSize",
     "clean": "rm -rf build; rm -rf transpiled; rm -rf types/generated",
     "clean-chain": "rm -rf blockchain && cp -r snapshots/0x-Kyber blockchain",
-    "compile": "truffle compile --all",
+    "compile": "./node_modules/.bin/truffle compile --all",
```

Comment:
any reason for this?

Reply:
i think this was to use the upgraded version of solc compiler as specified in the `truffle.js`

https://hackernoon.com/compile-solidity-smart-contracts-faster-in-truffle-projects-eb8bebb61ec3

i think before this, it was still trying to use the old version since truffle v5 by default still uses 0.4.25